src/utils/logger.ts

```diff
@@ -3,6 +3,13 @@ import RequestPipelineContext from '../request-pipeline/context';
 import { IncomingMessage, OutgoingHttpHeaders } from 'http';
 import { ServiceMessage } from '../typings/proxy';
 import RequestOptions from '../request-pipeline/request-options';
+import errToString from './err-to-string'
+
+function getIncorrectErrorTypeMessage (err: object) {
+    const errType = typeof err;
+
+    return `The "${errToString(err)}" error of the "${errType}" type was passed. Make sure that service message handlers throw errors correctly.`;
```

Comment:
I propose another error message.

```js
Incorrect "${errType}" error was passed to the error handler. Make sure that service message handlers throw errors correctly.
```

Reply:
Don't you want to show the error text if it exists?
for example:

```JS
throw 'string error';
```
`The "string error" error of the "string" type was passed. `

Reply:
Ok. Let it be.

Reply:
@DIRECTcut check this message please:
`The "${errToString(err)}" error of the "${errType}" type was passed. Make sure that service message handlers throw errors correctly.`

Reply:
It's unclear what `correctly` means. For example, does it mean:

`The "${errToString(err)}" error of the "${errType}" type was passed. Make sure that service message handlers throw errors of a correct type.`

Reply:
the error should be of Error type.
In JS you can make the following:
```JS
throw 1
```
but the correct usage is:
```JS
throw new Error('1');
```
What to do you think if I change it to
`Make sure that service message handlers throw errors of the Error type`